src/mito2/src/region.rs

```diff
@@ -60,7 +62,7 @@ impl RegionUsage {
 /// - Only the region worker thread this region belongs to can modify the metadata.
 /// - Multiple reader threads are allowed to read a specific `version` of a region.
 #[derive(Debug)]
-pub(crate) struct MitoRegion {
+pub struct MitoRegion {
```

Comment:
I'd recommend adding a new request (maybe a bulk load request) and handling it in the engine. We should not make it public.